ovms_env/bin/Activate.ps1

```diff
@@ -0,0 +1,247 @@
+<#

```

Comment:
please remove utility files and directories from pull request: make.save/ovms_env/rag_env etc. those are probably added by mistake? Please use .gitignore to ignore it automatically